openmdao_book/other_useful_docs/developer_docs/sphinx_decorators.ipynb

```diff
@@ -0,0 +1,127 @@
+{
```

Comment:
<p>At least in reviewnb, I see strange text in green boxes to the right of the auto generated docs. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Yeah this is just the new way we're doing it. The version is noted in the tabs

Reply:
<p>Yeah, the green boxes show what has changed. Similar to git diff</p>

Reply:
<p>Oh that's the markdown link name <code>![The Sphinx generated documentation for a method that is decorated but nothing fixingthe problem in the docstring](images/decorated_method_incorrect_docs.png)</code><span class="ql-cursor">﻿</span></p>